fedmsg_meta_umb/rpmdeplint.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# fedmsg_meta_umb is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg_meta_umb is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors: Gowrishankar Rajaiyan <grajaiya@redhat.com>
+
+import logging
+
+from fedmsg.meta.base import BaseProcessor
+
+
+log = logging.getLogger(__name__)
+
+
+class RPMDEPLINTProcessor(BaseProcessor):
+    topic_prefix_re = r'/topic/VirtualTopic\.eng'
+
+    __name__ = 'rpmdeplint'
+    __description__ = 'find errors in RPM packages in the context of their dependency graph'
+    __obj__ = 'rpmdeplint'
+    __docs__ = 'https://rpmdeplint.readthedocs.io/en/latest/'
```

Comment:
Can you please change this to https://docs.pagure.org/rpmdeplint/ ? I am trying to get away from the readthedocs stuff in favour of consolidating on pagure.